src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -76,7 +76,7 @@ object AstTools {
 
       astNode match {
         case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlStringType => WdlString(t.getSourceString)
-        case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString)
+        case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString, false)
```

Comment:
This does feel more like a subtype of something than a constructor arg supplied field FWIW